<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iniciar sesión</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Iniciar sesión</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="toolTip">
        <string>Cerrar</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="passwordLayout">
     <item>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="placeholderText">
        <string>Introduce tu contraseña</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="togglePasswordButton">
       <property name="toolTip">
        <string>Mostrar/ocultar contraseña</string>
       </property>
       <property name="text">
        <string>Ver</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="confirmLayout">
     <item>
      <spacer name="confirmSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Confirmar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="separatorLayout">
     <item>
      <widget class="Line" name="lineLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orLabel">
       <property name="text">
        <string>o</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold; padding: 8px; background-color: #e0e0e0;</string>
     </property>
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
